qshift/templates/quarkus-extensions/skeletons/src/main/java/MyEntity.java

```diff
@@ -0,0 +1,31 @@
+{% if values.enable_rest %}
+package org.acme;
```

Comment:
Why don't you set the package name according to the value `java_package_name` passed using the parameter ?

Reply:
done